### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by Recillah-Khamala**
For this pull request,

- I used JavaScript to ensure that emails submitted to the form are valid and the invalid information is not submitted.
- I also used CSS to style the changes and required HTML changes to make this possible.

### CODE REVIEW

File: js/pwindow.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+//   const button = document.createElement('button');
+//   const h1 = document.createElement('h1');
+//   const p = document.createElement('p');
+//   const a = document.createElement('a');
+
+//   // working on image in the card
+//   image.src = element.img;
+//   image.classList.add('cards-pic');
+//   image.setAttribute('alt', 'Portfolio first card');
+
```
Comment (1GORDON) [inline js/pwindow.js]: Great job team 🤝

- [ ] It is regarded as a bad practice according to the [Javascript good practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) to leave commented code in your project. Kindly remove commented and unused code from your javascript files.

File: index.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h2 class="first">I'm Recillah <br> Glad To See You!</h2>
 
         <!-- Middle section -->
         <section class="works" id="portfolio">
-            <div class="single-card card1">
+            <!-- <div class="single-card card1 ">
```
Comment (1GORDON) [inline index.html:79]: Great job team 🤝

- [ ] Kindly remove commented code from your HTML file. This is not regarded as a good practice.

File: index.html [hunk 3]
```diff
@@ -285,7 +285,7 @@ <h2 class="heading-2">Contact me</h2>
                             with it
                         </p>
                     </div>
-                    <form action="https://formspree.io/f/mgedjkyb" id="form1" method="post" class="contact-form">
+                    <form noValidate action="https://formspree.io/f/mgedjkyb" id="form1" method="post" class="contact-form">
```
Comment (1GORDON) [inline index.html:288]: Great job team 🤝

- [ ] As specified in the project requirements, your form is supposed to submit if the validation is okay. Unfortunately, your form is not submitted even when the email is lowercase. Kindly make sure your form is submitted properly to `form spree`.

File: index.html [hunk 4]
```diff
@@ -296,15 +296,18 @@ <h2 class="heading-2">Contact me</h2>
                             <textarea placeholder="Write your message here" id="message" name="message" maxlength="500"
                                 required></textarea>
```
Comment (1GORDON) [inline index.html:297]: Great design team 🤝

- [ ] [Optional] You may kindly make your text area similar to the design provided in the Figma template.
- Your design
form

- Figma design
Figma

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Rn486 & @elfin-git  ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] ⛔️ Kindly make sure your `linters` are passing. At the moment your `ESLint` is failing. It is hard to have a project approved when the `linters` are failing.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Rn486    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**